package.json

```diff
@@ -63,7 +64,7 @@
     "eslint-plugin-node": "^10.0.0",
```

Comment:
If I'm not mistaken, we can remove:
- eslint-plugin-standard
- eslint-plugin-promise
- eslint-plugin-node
- eslint-plugin-import
- eslint-config-standard
- eslint-config-semistandard